src/odemis/driver/test/symphotime_test.py

```diff
@@ -135,10 +151,163 @@ def callback(flow, array):
             # Start measuring, wait, then stop
             self.controller.data.subscribe(callback)
             self.assertTrue(self.controller.isMeasuring())
+            time.sleep(0.5)
```

Comment:
Why do you put a sleep _after_ the isMeasuring() ? Shouldn't it be before? Or is there a requirement between `subscribe()` and `unsubscribe()`?